bin/openstack-status

```diff
@@ -59,29 +58,12 @@ def parse_options(*args, **kwds):
                         dest='headless')
     parser.add_argument(
         '--version', action='version', version='%(prog)s {}'.format(version))
-    return parser.parse_args()
-
-
-def populate_config(opts):
-    presaved_config = os.path.join(
-        utils.install_home(), '.cloud-install/config.yaml')
-
-    # Always override presaved config if defined in cli switch
-    if opts.config_file and os.path.exists(opts.config_file):
-        _cfg = yaml.load(utils.slurp(opts.config_file))
-    # Pull from ~/.cloud-install/config.yaml if exists
-    elif os.path.exists(presaved_config):
-        _cfg = yaml.load(utils.slurp(presaved_config))
-    else:
-        _cfg = {}
-    # Update cfg dict with command line opts
-    _cfg.update(vars(opts))
-    return Config(cfg_obj=_cfg)
+    return parser.parse_args(argv)
 
 
 if __name__ == '__main__':
-    opts = parse_options(sys.argv)
-    config = populate_config(opts)
+    opts = parse_options(sys.argv[1:])
```

Comment:
other than keeping it similar to how `openstack-install` does its parsing of options